coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -2068,10 +2068,21 @@ def hardtanh(context, node):
 
 @register_torch_op(torch_alias=["concat"])
 def cat(context, node):
-    inputs = _get_inputs(context, node)
+    inputs = _get_inputs(context, node, min_expected=1)
+
+    xs = inputs[0]
+    # PyTorch can have empty tensor, which is then ignored
+    # However, CoreML does not allow such empty tensor, so remove them now
+    if np.any([x.rank > 1 or x.shape[0] > 0 for x in xs]):
+        xs = [x for x in xs if x.rank > 1 or x.shape[0] > 0]
```

Comment:
I never used `map` before, how to use it here?

Reply:
I think you might can use `map` along with `filter`

Reply:
Here is the doc on `map`:
https://www.w3schools.com/python/ref_func_map.asp
You can use a `lambda` statement rather than a named function.

Henry is right you would also need to use `filter`. So that probably makes things for complicated than it's worth. I guess this isn't a good application of `map`.